docs/docsite/rst/reference_appendices/special_variables.rst

```diff
@@ -79,14 +90,18 @@ play_hosts
 ansible_play_name
     The name of the currently executed play. Added in ``2.8``.
 
+play_role_names
```

Comment:
don't create a new variable that is not conforming to the `ansilbe_` pattern

Reply:
Hm, this seems to be an error introduced by rebasing, will correct.

Reply:
This should have been corrected with the last commit.